README.md

```diff
@@ -240,7 +240,7 @@ client.setHttpClient(adyenHttpClientWithProxy);
 If your proxy requires authentication, set all the settings as system properties instead (don't mix with previous approach), for example: 
 
 ~~~~ java
-System.setProperty("https.proxyHost", "129.0.0.1");
+System.setProperty("https.proxyHost", "129.0.1.1");
```

Comment:
@DjoykeAbyah this is interesting, automation changed the proxy as it is coincidentally the exact same number as the version number.. Let's change it back so we do not keep bumping the proxy along with the version :)

Reply:
I have corrected this manually. We need to look at the version bump logic and address it